src/core/components/courses/TrainerMissionTable.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <div v-if="!!trainerMissions.length"
+    class="q-mb-xl">
+    <p class="text-weight-bold">Ordres de mission</p>
+    <div class="q-mb-sm">
+      <q-card>
+        <ni-responsive-table :data="trainerMissions" :columns="columns" v-model:pagination="pagination"
+          :loading="loading" :rows-per-page-options="[5, 10, 15, 20]" :hide-bottom="false">
+          <template #body="{ props }">
+            <q-tr :props="props">
+              <q-td v-for="col in props.cols" :key="col.name" :data-label="col.label" :props="props"
+                :style="col.style" :class="[{ 'border': props.rowIndex === 0 }]">
+                <template v-if="col.name === 'actions'">
+                  <ni-button icon="file_download" color="primary" type="a" :href="props.row.file.link"
+                    :disable="!props.row.file.link" />
+                </template>
+                <template v-if="col.name === 'courses'">
+                  <div v-for="course of props.row.courses" :key="course._id" class="q-mb-xs course">
+                    <router-link class="clickable-name" :to="gotToCourse(course._id)">
+                      {{ composeCourseName(course, true) }}
+                    </router-link>
+                  </div>
+                </template>
+                <template v-else>{{ col.value }}</template>
+              </q-td>
+            </q-tr>
+          </template>
+        </ni-responsive-table>
+      </q-card>
+    </div>
+  </div>
+  <div v-else class="text-italic">Pas d'ordres de mission</div>
+</template>
+
+<script>
+import { ref } from 'vue';
+import Button from '@components/Button';
+import ResponsiveTable from '@components/table/ResponsiveTable';
+import CompaniDate from '@helpers/dates/companiDates';
+import { DD_MM_YYYY } from '@data/constants';
+import { formatPrice } from '@helpers/utils';
+import { composeCourseName } from '@helpers/courses';
+
+export default {
+  name: 'TrainerMissionTable',
+  props: {
+    loading: { type: Boolean, default: false },
+    trainerMissions: { type: Array, default: () => [] },
+  },
+  components: {
+    'ni-button': Button,
+    'ni-responsive-table': ResponsiveTable,
+  },
+  emits: ['refresh'],
+  setup () {
+    const pagination = ref({ rowsPerPage: 5, sortBy: 'date', descending: true });
+    const columns = ref([
+      {
+        name: 'date',
+        label: 'Date',
+        align: 'left',
+        field: 'date',
+        format: value => CompaniDate(value).format(DD_MM_YYYY),
+      },
+      {
+        name: 'courses',
+        label: 'Formation',
+        align: 'left',
+        field: 'courses',
+        style: 'width: 70%',
```

Comment:
Pourquoi mettre 70% ici ? L'affichage est reduit sur telephone

Reply:
oui c'est très moche sur ordi sans ça.  les colonnes font toutes la même taille et sont démesurées par rapport à la colonne formations

Reply:
Et est-ce qu'on peut pas s'en sortir en definissant :  style: 'course' dans la colonne et en definissant une largeur pour les grands ecrans `@media screen and (min-width: 768px)` ?

Reply:
on peut pas mettre une classe dans un style